docs/OV_Runtime_UG/Python_API_exclusives.md

```diff
@@ -0,0 +1,143 @@
+# OpenVINO™ Python API exclusives {#openvino_docs_OV_Runtime_UG_Python_API_exclusives}
+
+OpenVINO™ Runtime Python API is exposing additional features and helpers to elevate user experience. Main goal of Python API is to provide user-friendly and simple, yet powerful, tool for Python users.
+
+## Easier model compilation 
+
+`CompiledModel` can be easily created with the helper method. It hides `Core` creation and applies `AUTO` device by default.
+
+@sphinxdirective
+
+.. doxygensnippet:: docs/snippets/ov_python_exclusives.py
+    :language: python
+    :fragment: [auto_compilation]
+
+@endsphinxdirective
+
+## Model/CompiledModel inputs and outputs
+
+Besides functions aligned to C++ API, some of them have their Pythonic counterparts or extensions. For example, `Model` and `CompiledModel` inputs/outputs can be accessed via properties.
+
+@sphinxdirective
+
+.. doxygensnippet:: docs/snippets/ov_python_exclusives.py
+    :language: python
+    :fragment: [properties_example]
+
+@endsphinxdirective
+
+Please refer to Python API documentation on which helper functions or properties are available for different classes.
```

Comment:
Refer to Python API documentation on which helper functions or properties are available for different classes.

Reply:
Done